nautobot_netbox_importer/diffsync/models/extras.py

```diff
@@ -306,6 +306,8 @@ class Status(ChangeLoggedModelMixin, NautobotBaseModel):
     _attributes = ("slug", "name", "color", "description", *ChangeLoggedModelMixin._attributes)  # TODO content_types?
     _nautobot_model = extras.Status
 
+    _ignored_fields = NautobotBaseModel._ignored_fields | {"destination_for_associations", "source_for_associations"}
```

Comment:
good catch, this is because it is running for instantiating both NetBox and Nautobot models, I will figure out how to move this into NetBox only loading.